### PR TITLE
Added Dockerfile and docker-compose for live reload during development

### PR DESCRIPTION
**Pull request by this-Aditya**

### CODE REVIEW

File: docker-compose.live-reload.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  sep-ui-live-dev:
+    build:
+      context: .
+      dockerfile: ./Dockerfile.dev
+    ports:
+      - '3000:3000'
+    volumes:
+      - .:/app
+    environment:
+      - PORT=3000
```
Comment (yatharthranjan) [inline docker-compose.live-reload.yml:12]: Perhaps set this on the run command in Dockerfile, so the app works as expected even when not running through docker-compose.

File: Dockerfile.dev [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+WORKDIR /app
+
+COPY package*.json ./
+
+RUN npm install
+
+EXPOSE 3000
+
+CMD ["npm", "run", "dev"]
```
Comment (yatharthranjan) [inline Dockerfile.dev:13]:
```suggestion
CMD ["npm", "run", "dev", "--", "-p", "3000"]
```

**Review by yatharthranjan — COMMENTED** [review]
looks good, a minor suggestion

File: docker-compose.live-reload.yml (hunk 1 above)
Comment (this-Aditya) [reply docker-compose.live-reload.yml]: Hi Yatharth,

I have moved the environment variable settings to the Dockerfile, but I think we still can’t start it directly from the Dockerfile. It also needs to reference the host directory, and I am just copying the package.json into the Dockerfile. Everything else is used in bind mounts.

**Review by yatharthranjan — APPROVED** [review]
LGTM, thanks!

**Pull request merged**